pkg/minikube/download/iso.go

```diff
@@ -142,3 +143,85 @@ func downloadISO(isoURL string, skipChecksum bool) error {
 
 	return download(urlWithChecksum, dst)
 }
+
+func WindowsISO(windowsIsoURL string) error {
```

Comment:
should probably be `downloadWindowsISO`, but since the function does not differ much with downloadISO, look for ways to merge the 2

Reply:
Merged these two in the latest push